repo-projects/Directory.Build.props

```diff
@@ -106,6 +106,8 @@
 
   <!-- Common args used for building and testing. -->
   <PropertyGroup>
+    <CommonArgs Condition="'$(DotNetBuildSourceOnly)' != 'true'">$(CommonArgs) $(FlagParameterPrefix)productBuild</CommonArgs>
```

Comment:
@mmitche does this look right to you? We could also always unconditionally pass this switch in as source-build implies product build. We are following this infra: https://github.com/dotnet/arcade/blob/6b63184d8ff276115846b232816ece935a26a22c/eng/common/build.sh#L132-L144

Reply:
Yeah, I think you would only need one switch. SB implies PB